### DISCUSSION TITLE
Bump gin-gonic/gin version to 1.9.1

### DISCUSSION THREAD

**Discussion by paulohenriquesg** [getsentry/sentry-go#676] [Q&A] [answered]
Hi guys! 

I got some reports from BlackDuck that the `gin-gonic/gin` lib needed to be updated to their latest version.

Do you have any plans on performing this update?

Thank you!

**Comment by tonyo** [comment, marked as answer]
Hi @paulohenriquesg ,

No plans to bump it in sentry-go in the near future, but you are absolutely encouraged to update it in your end application. 
We described our approach to bumping dependencies here: https://github.com/getsentry/sentry-go/security/policy#dependency-update-policy
